package-lock.json

```diff
@@ -16672,6 +16672,27 @@
         }
       }
     },
+    "packages/identity-agent/node_modules/@web5/api/node_modules/@web5/dids": {
+      "version": "0.2.3",
```

Comment:
Is it OK that it's pulling in this version of `@web5/dids` vs `0.2.4` like the others?

Reply:
Great catch!  

NPM messed that up.  Had to blow away `package-lock.json` and rebuild.  Resolved.